pilot/pkg/xds/debug.go

```diff
@@ -139,69 +140,116 @@ func (s *DiscoveryServer) InitDebug(mux *http.ServeMux, sctl *aggregate.Controll
 		ServiceDiscovery: s.MemRegistry,
 		Controller:       s.MemRegistry.Controller,
 	})
-	s.AddDebugHandlers(mux, enableProfiling, fetchWebhook)
+	internalMux := http.NewServeMux()
```

Comment:
We could have stored in internalMux in Server struct so that we do not have to pass it around